xbmc/video/windows/GUIWindowVideoBase.cpp

```diff
@@ -897,10 +897,12 @@ void CGUIWindowVideoBase::GetContextButtons(int itemNumber, CContextButtons &but
         buttons.Add(CONTEXT_BUTTON_PLAY_PARTYMODE, 15216); // Play in Partymode
       }
 
-      //if the item isn't a folder or script, is a member of a list rather than a single item
-      //and we're not on the last element of the list,
-      //then add add either 'play from here' or 'play only this' depending on default behaviour
-      if (!(item->m_bIsFolder || item->IsScript()) && m_vecItems->Size() > 1 && itemNumber < m_vecItems->Size()-1)
+      //if the item isn't a folder or script, is not explicitly marked as not playable,
+      //is a member of a list rather than a single item and we're not on the last element of the list,
+      //then add either 'play from here' or 'play only this' depending on default behaviour
+      if (!(item->m_bIsFolder || item->IsScript())
+          && (!item->HasProperty("IsPlayable") || item->GetProperty("IsPlayable").asBoolean())
```

Comment:
Condition is also wrong. HasProperty("IsPlayable") || something, will evaluate to true if the first condition matches. Means listitem.seProperty("isPlayable", "false") will enable the context item too

Reply:
Note that `HasProperty` is negated. At the moment I believe the context item is added only when IsPlayable is explicitly set to false.

Reply:
You're right, missed the negation somehow :) think I need either to stop using the phone or buy one with a bigger screen.

Either way I think it's better to rewrite the second condition to the negation too. IsPlayable, afaik is just a property we can use from plugins to signal Kodi the item is playable. It is not used at a large extension (if at all) throughout the core .

Thanks for the contribution